src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke.tasks;
+
+/**
+ * Bare task that has a task name and a boolean condition on whether the task is done.
+ */
+public class Task {
+    protected String taskName;
```

Comment:
Having task in task name seems abit redundant, wouldn't it be better to shorten it to name?